api/src/wfl/service/firecloud.clj

```diff
@@ -11,6 +11,18 @@
            [java.util UUID]
            [wfl.util UserException]))
 
+(defn terra-ui-url
```

Comment:
We use these URL helpers to generate links in Slack notifications, but do not interact with the corresponding services programmatically.

To move quickly I pulled them into `firecloud` since that's the service powering Terra, but I'd move these somewhere else if preferred.